src/controllers/dynakube/activegate/internal/statefulset/agbuilder/builder_test.go

```diff
@@ -0,0 +1,81 @@
+package agbuilder
```

Comment:
would it make sense to move the contents of the `agbuilder` package to the `activegate` package? `agbuilder` is a weird name for a package